### ISSUE TITLE
Sidebar is not updating as expected

### ISSUE THREAD

**Issue by RaiBnod** [NubeIO/rubix-ce#1319]
Issues right now:

- When we add/edit/delete Views/Network/Device it needs to be reflected on the sidebar as well, but we need to reload the page to see that on the sidebar

**Comment by nguyenoanh5597** [comment]
hi @riteshakya037 I think it is also related to this issue: https://github.com/NubeIO/rubix-ce/issues/1308

**Comment by riteshakya037** [comment]
@nguyenoanh5597 Yes this issue you linked would definitely be a subset of this. We do have a lot more sync issues, which this ticket should cover. Thanks for the heads up mate.

**Comment by nguyenoanh5597** [comment]
I fixed issue https://github.com/NubeIO/rubix-ce/issues/1308, and also fixed the same for devices/views menu, hope [this PR](https://github.com/NubeIO/rubix-ce/pull/1310) will be useful to you

**Comment by riteshakya037** [comment]
@nguyenoanh5597 Yes absolutely life saver. Lets just consider the one you are using as concrete and mark this as duplicate. One slight suggestion regarding Views, we already have a react context that refreshes the sidebar items based on changes. Please migrate to using that so as to keep it consistent. 

- Use ViewContext's context by adding 
```
  const { updateViewContext } = useContext(ViewContext);
```

- Whenever there in a change call the following piece of code, it will refresh the appropriate view listing

```
 updateViewContext({
          id: generateUUID(),
          locUUID,
          netUUID,
          hostUUID,
        });
```
